src/EventSubscriber/PageHeaderSubscriber.php

```diff
@@ -39,19 +53,37 @@ public function setPageHeader(PageHeaderDisplayEvent $event) {
     }
 
     $overview = $node->localgov_guides_parent->entity ?? NULL;
-    if (!empty($overview)) {
-      $event->setTitle($overview->getTitle());
-      if ($overview->get('body')->summary) {
-        $event->setLede([
-          '#type' => 'html_tag',
-          '#tag' => 'p',
-          '#value' => $overview->get('body')->summary,
-        ]);
+    if ($this->settings->get('legacy_header')) {
```

Comment:
If a site hasn't explicity set to use the legacy header, this will automatically opt them into the new style until they save the config form. Maybe consider reversing the check to see if they want new style headers? Or make sure to include an update hook for existing sites to set this as legacy header. Note the settings form below does assume legacy header as default, but it won't apply until its saved.

Reply:
Thanks, I missed that (saved the form before ever testing). I've taken your first suggestion to require the checkbox to be set to use the 'modern' rendering. That means the default, unsaved state uses the existing rendering (since it defaults to `FALSE`).